services/explorer/src/explorer/templates/partials/_timeline_chart.html

```
@@ -0,0 +1,85 @@
+<div class="d-flex justify-content-between align-items-center mb-2">
+  <div class="btn-group btn-group-sm" id="timeline-bucket-toggle">
+    {% for b in ["day", "week", "month"] %}
+    <a class="btn btn-outline-secondary {% if bucket == b %}active{% endif %}"
+       hx-get="/analytics/partials/timeline?days={{ days }}&bucket={{ b }}"
+       hx-target="#timeline-chart-area"
+       hx-swap="innerHTML"
+       href="#">{{ b | capitalize }}</a>
+    {% endfor %}
+  </div>
+  <div class="btn-group btn-group-sm" id="timeline-metric-toggle">
+    <button class="btn btn-outline-secondary active" onclick="setTimelineMetric('plays', this)">Plays</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)